workflows/files/ZWECONF.properties

```diff
@@ -30,13 +38,21 @@ zowe_setup_dataset_parmlib=IBMUSER.ZWEV2.CUST.PARMLIB
 #  JCL library where Zowe will store temporary JCLs during initialization
 zowe_setup_dataset_jcllib=IBMUSER.ZWEV2.CUST.JCLLIB
 
+# zowe_setup_dataset_loadlibPlugin
+# Label: Zowe setup Utilities and extensions library
+# Abstract: Load library where Zowe will store libraries Utilities for use by Zowe and extensions
```

Comment:
Load library where Zowe stores libraries for Utilities for use by Zowe and extensions